assets/src/stories-editor/plugins/export.js

```diff
@@ -92,6 +92,8 @@ const handleExport = ( { postId, createErrorNotice, createSuccessNotice, removeN
 
 export const name = 'amp-story-export';
 
+export const icon = 'media-archive';
```

Comment:
What does this do?

Reply:
Define the icon that is used when registering the plugin here:

https://github.com/ampproject/amp-wp/blob/50bc094b83764fa3409b60694606c1660f24f994/assets/src/stories-editor/index.js#L306-L312